x-pack/plugins/apm/public/components/app/help_popover/help_popover.scss

```diff
@@ -0,0 +1,9 @@
+.apmHelpPopover__panel {
+  max-width: $euiSize * 30;
+}
+
+.apmHelpPopover__content {
+  @include euiYScrollWithShadows;
+  max-height: 40vh;
+  padding: $euiSizeS;
+}
```

Comment:
Can you add this as styled-components instead of sass?

Reply:
@sqren Updated here https://github.com/elastic/kibana/pull/103759/commits/ed1c71f97fbfe8a256b4159deac2ff8177489cd9